tests/clients/test_basic_client.py

```diff
@@ -60,9 +60,11 @@ def test_metrics_reporter_fit() -> None:
 def test_metrics_reporter_evaluate() -> None:
     test_current_server_round = 2
     test_loss = 123.123
-    test_metrics: Dict[str, Union[bool, bytes, float, int, str]] = {"test_metric": 1234}
+    test_metrics: Dict[str, Scalar] = {"test_metric": 1234}
+    test_metrics_testing: Dict[str, Scalar] = {"testing_metric": 1234}
+    test_metrics = {"test_metric": 1234, "testing_metric": 1234, "test - loss": 123.123}
```

Comment:
It looks like you're immediately redefining test metrics here from above? I think you can at least delete that.

Reply:
I made a mistake with this change, for some reason test_metrics is modified in place, so I can only add a new one at the end to compare with, ill change the name in the next commit